### ISSUE TITLE
nix-shell MacOSX git checkout

### ISSUE THREAD

**Issue by coretemp** [NixOS/nixpkgs#32981]
## Issue description
How can I do a git checkout using nixpkgs (17.09 version) git on MacOSX? 

```
#! /usr/bin/env nix-shell
#! nix-shell -I nixpkgs=./nixpkgs --pure -i sh -p cacert git
SSL_CERT_FILE=????? git clone https://github.com/NixOS/nixpkgs.git 
```
Solve for ?????

returns something like 

```
fatal: unable to access 'https://github.com/NixOS/nixpkgs.git': error setting certificate verify locations:
```

**Comment by LnL7** [comment]
Do you have cacert installed in the default profile? I'm pretty sure openssl looks for `/nix/var/nix/profiles/default/etc/ssl/certs/ca-bundle.crt` if non of the variables are set.
Strangely it works for me even tho that doesn't exist on my machine.

**Comment by coretemp** [comment]
It has been fixed in master, but not in 17.09, which is what this issue was about.
